tmail-backend/pom.xml

```diff
@@ -359,6 +359,11 @@
                 <artifactId>firebase-admin</artifactId>
                 <version>${firebase-admin-sdk.version}</version>
             </dependency>
+            <dependency>
+                <groupId>net.javacrumbs.json-unit</groupId>
```

Comment:
Why this version do not come from james managed POM?

Reply:
It does not exist in James 
We can adapt James later, then update it in next update james sha1